src/Widgets/TerminalWidget.vala

```diff
@@ -103,6 +102,32 @@ namespace PantheonTerminal {
             private set;
         }
 
+        public double _zoom_factor = 1.0;
+        public double zoom_factor {
+            get {
+                return _zoom_factor;
+            }
+            set {
+                _zoom_factor = value;
+
+                if (_zoom_factor < 0.19) {
+                    _zoom_factor = 0.2;
+                } else if (_zoom_factor > 5.01) {
+                    _zoom_factor = 5;
+                }
+
```

Comment:
Because "zoom_factor" property is bound to a setting, it feels better to test value and assign _zoom_factor accordingly rather than assign _zoom_factor and then maybe immediately reassign it - i.e.

```
                if (value < 0.19) {
                    _zoom_factor = 0.2;
                } else if (value > 5.01) {
                    _zoom_factor = 5;
                } else {
                    _zoom_factor = value;
                }
```

(Although no adverse effects were noticed your way)